web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -83,7 +95,7 @@ h3, h4,
 
 #site-header {
     background: none;
-    border-image-source: url(images/tile-y-border.jpg);
+    border-image-source: url(images/tile-y.jpg);
```

Comment:
Nope, you're absolutely right. At that size it doesn't really matter if it's a retina asset or not. I guess I was thinking mostly of the unnecessary extra request for the `@1x` image, but not worth bothering with really.